packages/flow-core/src/components/f-suggest/f-suggest.ts

```diff
@@ -201,10 +220,23 @@ export class FSuggest extends FRoot {
 		return Array.isArray(this.suggestions);
 	}
 
+	get isSuggestionObjects() {
+		return (
+			this.isSuggestionArray &&
+			(this.suggestions as FSuggestTemplate[])?.every(
+				item => typeof item === "object" && item !== null && !Array.isArray(item)
+			)
+		);
+	}
+
 	get filteredSuggestions() {
 		if (this.value) {
-			if (this.isSuggestionArray) {
+			if (this.isSuggestionArray && !this.isSuggestionObjects) {
 				return (this.suggestions as string[])?.filter(sg => sg.includes(this.value as string));
+			} else if (this.isSuggestionObjects) {
```

Comment:
you need to check if suggestions is array objects and its has template, toString adn value property present.

Otherwise it will never execute last else part